### ISSUE TITLE
Broken links in report

### ISSUE THREAD

**Issue by jinjor** [reg-viz/reg-cli#184]
I simply ran this command

```bash
$ reg-cli actual/ expected/ diff -R index.html
```

and found some links are broken.

![image](https://user-images.githubusercontent.com/2568148/41833044-b0cb1294-7889-11e8-8658-b3476808f091.png)

The directly structure I'm seeing is like this.

![image](https://user-images.githubusercontent.com/2568148/41833560-ba2a61c6-788b-11e8-8809-c55e5d02260d.png)

----

Another attempt is like this. I added `/` at the end of `diff`

```bash
$ reg-cli actual/ expected/ diff/ -R index.html
```

and got this.

![image](https://user-images.githubusercontent.com/2568148/41833917-3aa3335e-788d-11e8-9d7e-c12571c2aca0.png)

And results in this directly structure.

![image](https://user-images.githubusercontent.com/2568148/41833887-1decb618-788d-11e8-98e1-bbf63eade829.png)

----

- macOS
- node v8.9.4
- reg-cli (globally installed today)

**Comment by bokuweb** [comment]
@jinjor Thanks for your report ;) I'll do some investigation. Please wait a moment.

**Comment by bokuweb** [comment]
@jinjor I published 	v0.11.5 🎉

**Comment by jinjor** [comment]
I just confirmed the latest version works well.
Thanks for the quick fix! 😃
